### PR TITLE
Auto updater fix

### PR DESCRIPTION
**Pull request by jdent02**
Suggest we do a release for this fix.  That way the auto updater should catch it even though it's kinda broken right now.

### CODE REVIEW

**Comment by bsavery** [comment]
Cool.  21.4 is coming out monday..

**Comment by jdent02** [comment]
Okay.  FYI, I set this up to only allow updating from releases and the master branch.  The other branches could have been included but I figured that would cause more confusion than it's worth.  Anybody who wants to test the other branches will still have to manually download and install.

File: addon_updater.py [hunk 1]
```diff
@@ -76,7 +76,9 @@ def __init__(self):
 		self._tag_latest = None
 		self._tag_names = []
 		self._latest_release = None
-		self._include_master = True
+		self._include_branches = False
+		self._include_branch_list = ['master']
```
Comment (bsavery) [inline addon_updater.py]: So these two lines seem contradictory include_branches is is false but there's a list of branches to include?

**Comment by jdent02** [comment]
The preferences override that setting (they start at line 922 in the _ops module).

If you set the preferences to allow branches but don't explicitly define them then it defaults to only showing the 'master' branch

**Pull request merged**